pkg/systemd/manifest.json.in

```diff
@@ -9,21 +9,45 @@
     "menu": {
         "index": {
             "label": "System",
-            "order": 10
+            "order": 10,
+            "keywords": {
+                "general": {
+                    "matches": ["time", "date", "restart", "shut", "domain", "machine", "operating system", "os", "asset tag", "ssh", "power", "version", "host"]
+                },
+                "hw": {
+                    "matches": ["hardware", "mitigation", "pci", "memory", "cpu"],
+                    "goto": "/system/hwinfo"
+                }
+            }
         },
         "services": {
             "label": "Services",
-            "order": 100
+            "order": 100,
+            "keywords": {
+                "general": {
+                    "matches": ["service", "systemd", "target", "socket", "timer", "path", "unit", "systemctl"]
```

Comment:
+ restart

perhaps + boot?

Reply:
how does restart and boot fit into services page?